scipy/signal/filter_design.py

```diff
@@ -4843,6 +4845,206 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def _hz_to_erb(hz):
+    """
+    Utility for converting from frequency (Hz) to the
+    Equivalent Rectangular Bandwith (ERB) scale
+    ERB = frequency / EarQ + minBW
+    """
+    EarQ = 9.26449
+    minBW = 24.7
+    return hz / EarQ + minBW
+
+
+def gammatone(freq, ftype, order=None, numtaps=None, fs=None):
+    """
+    Gammatone filter design.
+
+    This function computes the coefficients of an FIR or IIR gammatone
+    digital filter [1].
```

Comment:
```suggestion
    digital filter [1]_.
```
As is, this will not render properly, see CircleCI output (please check to make sure you're otherwise happy with it):

https://19633-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.signal.gammatone.html#scipy.signal.gammatone

Reply:
Fixed.